unit-tests/live/metadata/test-timestamps.py

```diff
@@ -0,0 +1,74 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+# test:device each(D400*)
+
+import pyrealsense2 as rs
+from rspy import test, log
+
+# This test is checking that timestamps of depth and infrared frames are consistent
+
+# Tolerance for gap between depth and color
+GLOBAL_TS_TOLERANCE = 1  # in ms
+FRAME_TS_TOLERANCE = 100  # in microseconds
+with test.closure("Test Timestamps Consistency"):
+    device, ctx = test.find_first_device_or_exit()
+    cfg = rs.config()
+    cfg.enable_stream(rs.stream.depth)
+    cfg.enable_stream(rs.stream.infrared, 1)
+    cfg.enable_stream(rs.stream.infrared, 2)
+    cfg.enable_stream(rs.stream.color)
+
+    pipe = rs.pipeline(ctx)
+    pipe.start(cfg)
+
+    try:
+        for _ in range(50):
+            frames = pipe.wait_for_frames()
+            depth_frame = frames.get_depth_frame()
+            ir1_frame = frames.get_infrared_frame(1)
+            ir2_frame = frames.get_infrared_frame(2)
+            color_frame = frames.get_color_frame()
+
+            if not (depth_frame and ir1_frame and ir2_frame):
+                log.e("One or more frames are missing")
+                continue
+
+            # Global timestamps
```

Comment:
please add comment that global time will be enabled becasue hw reset is done before